src/components/ProjectCard/Section/ProjectSection.styles.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  margin-bottom: 1rem;
+  gap: 1rem;
+`;
+
+export const Header = styled.div`
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  gap: 0.25rem;
+`;
+
+export const Title = styled.h2`
+  font-size: 1.5rem;
+  font-weight: bold;
+`;
+export const Subtitle = styled.p`
+  max-width: 50rem;
+  text-align: center;
```

Comment:
```suggestion
  text-align: center;
  padding: 0 1rem;
```
In smaller screens (like S10 or iPhone 11 Pro), the subtitle was too close to the edge, adding a padding should fix it
![image](https://github.com/user-attachments/assets/299a2100-9a81-4c50-a1c0-20934d3b92c6)